src/Magnum/GL/Test/BufferGLTest.cpp

```diff
@@ -138,15 +142,14 @@ void BufferGLTest::constructFromData() {
     CORRADE_COMPARE(c.size(), 5*4);
     CORRADE_COMPARE(d.size(), 5*4);
 
-    /** @todo How to verify the contents in ES? */
-    #ifndef MAGNUM_TARGET_GLES
-    CORRADE_COMPARE_AS(Containers::arrayCast<Int>(a.data()),
+    #ifndef MAGNUM_TARGET_GLES2
+    CORRADE_COMPARE_AS(DebugTools::bufferData<Int>(a),
```

Comment:
In the anticipation of the template removal, can you keep this as 

```suggestion
    CORRADE_COMPARE_AS(Containers::arrayCast<Int>(DebugTools::bufferData(a)),
```

and same elsewhere?